README.md

```diff
@@ -83,6 +104,13 @@ Install the lambda using sceptre:
 sceptre --var "profile=my-profile" --var "region=us-east-1" launch prod/lambda-template.yaml
```

Comment:
"curl https://s3.amazonaws.com/bootstrap-awss3cloudformationbucket-19qromfd235z9/lambda-template/master/lambda-template.yaml"

Might as well switch to [new-style S3 URLs](https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/) while you are doing a PR

Reply:
fixed